### ISSUE TITLE
Manage logging

### ISSUE THREAD

**Issue by alexbriggs** [chris48s/arcgis2geojson#70]
Hey Team! 🙂

 

I've been using your package regularly and have found it very useful. I've just come across a case where I would like to suppress your log statements as I'm converting 27k+ shapes which all raise the below warning message.

 

https://github.com/chris48s/arcgis2geojson/blob/937da747115a8525e38e60b887e2eb01d2ef1eae/arcgis2geojson/__init__.py#L285-L292

 

I'm running this on a timer function which means I get 27k+ log messages every hour which is clogging up my logs and making it hard to pick out other important messages. I've had a look and can suppress your logs using the below code.

 

 

```
import logging
logging.getLogger().setLevel(logging.ERROR)
```
However, it looks like you are just using the root logger in your code which means using the above could affect other loggers that are present as it sets it at the root level. Could you please update to use a non-root logger within your code so that it makes log handling easier when used in other packages?

 

Example would be:

 

 

```
import logging
logger = logging.getLogger("arcgis2geojson")
logger.warning("Object converted in non-standard crs - " + str(arcgis["spatialReference"])
```
This would be a very beneficial inclusion so thank you in advance 🙂

**Comment by chris48s** [comment]
I've just released version 3.0.3 which includes this fix

**Comment by alexbriggs** [comment]
Awesome, thanks a lot!!
